routers/web/repo/repo.go

```diff
@@ -599,6 +599,13 @@ func SearchRepo(ctx *context.Context) {
 
 	results := make([]*api.Repository, len(repos))
 	for i, repo := range repos {
+		if err = repo.GetOwner(ctx); err != nil {
+			ctx.JSON(http.StatusInternalServerError, api.SearchError{
+				OK:    false,
+				Error: err.Error(),
+			})
+			return
+		}
```

Comment:
Why is this needed? `repo_model.SearchRepository(opts)` initializes the owner property.